src/Sylius/Bundle/CoreBundle/Fixture/Factory/PromotionExampleFactory.php

```diff
@@ -170,4 +196,17 @@ private static function getCouponNormalizer(?FactoryInterface $couponFactory): \
             return $coupons;
         };
     }
+
```

Comment:
```suggestion
    /** @return iterable<string> */
```
and remove the entry from the `phpstan-baseline.neon`

Reply:
When I added this code, I received errors from Psalm

Reply:
What kind of errors?

Reply:
Psalm wanted me to use `Generator` in the annotation, but what it actually meant was that the `getCode` method could return a null value. Fixed.